covbot.py

```diff
@@ -597,6 +599,25 @@ def _short_location(self, location: str, length=int(12)) -> str:
         c.format = "org.matrix.custom.html"
         await e.respond(c, markdown=True, allow_html=True)
 
+    @command.new('risks', help=HELP['risks'][1])
+    @command.argument("risks", pass_raw=True, required=True)
+    async def risks_handler(self, event: MessageEvent, age: int) -> None:
+        if age < 0 or age > 110:
+            await self._respond(event, "Age must be between 0 and 110")
+            return
+        death_rate = -0.0012822 + (0.00000334593 * age ** 2) + ((2.7762 * 10 ** -15) * age ** 7)
+        intensive_care_rate = (1 / math.pi) + -0.358209 / (1 + 0.0827213 * math.exp(0.035025 * age))
+        hospitalization_rate = -0.0353956 - age * -0.00576217
+        survival_rate = 1 - death_rate
+        pf = lambda x: round(x * 100)
+        response = """
```

Comment:
This gives an interestingly-indented message:

![covbotrisks](https://user-images.githubusercontent.com/7524620/77587159-47d31680-6edf-11ea-8cad-74a2f8768d37.jpg)


I would suggest not indenting for consistency in output with `!cases`, but that's @pwr22's call overall.

Reply:
yup it was redneckly formatted. fixed that !